devops/cts_exclude_filter

```diff
@@ -0,0 +1,22 @@
+context
```

Comment:
This is CUDA only list at this point, right?
We may need a list per target?

Reply:
Yes.
Yes.